### PR TITLE
Add dataverse extension

### PR DESCRIPTION
**Pull request by yarikoptic**
attn @mih @bpoldrack @adswa 

WDYT?

### CODE REVIEW

**Comment by yarikoptic** [comment]
ok -- it isn't on pypi yet? (even her young brother gooey is there already!)

**Comment by adswa** [comment]
So much to do, so little time. I think we'll put it on a lab hackathon agenda.

**Comment by adswa** [comment]
Fixed the conflicts and rebased. A release is coming this evening.

File: extensions.yaml [hunk 1]
```diff
@@ -37,6 +37,10 @@ extensions:
     extra_setup_steps:
     - name: Install singularity
       run: scripts/ci/install-singularity.sh
+  datalad_dataverse:
+    cron_time: 25 02 * * *
+    ci: AppVeyor
+    ci-user: mih
```
Comment (mih) [inline extensions.yaml:43]:
```suggestion
    ci-user: mih
    branch: main
```

**Comment by mih** [comment]
Thanks for putting this together. The release was made. It is all green now, but very few tests actually run, because a dataverse token spec is missing. I don't know how to achieve that.

**Comment by adswa** [comment]
I can confirm its not enough to set the token as a repository secret

**Comment by yarikoptic** [comment]
cool -- it ran dataverse ones and they passed, let's proceed!

File: README.md [hunk 2]
```diff
@@ -12,31 +12,17 @@ This is a "dashboard" of various CIs of DataLad, its extensions, and underlying
...
-   [![Appveyor maint](https://ci.appveyor.com/api/projects/status/github/datalad/datalad?branch=maint&svg=true)](https://ci.appveyor.com/project/mih/datalad/branch/maint)
-   [![Appveyor master](https://ci.appveyor.com/api/projects/status/github/datalad/datalad?branch=master&svg=true)](https://ci.appveyor.com/project/mih/datalad/branch/master)
+   [![Travis tests status](https://travis-ci.com/datalad/datalad.png?branch=master)](https://travis-ci.com/datalad/datalad)
+   [![Build status](https://ci.appveyor.com/api/projects/status/github/datalad/datalad?branch=master&svg=true)](https://ci.appveyor.com/project/mih/datalad/branch/master)
    [![Documentation](https://readthedocs.org/projects/datalad/badge/?version=latest)](http://datalad.rtfd.org)
-   [![Extensions](https://github.com/datalad/datalad/actions/workflows/test_extensions.yml/badge.svg)](https://github.com/datalad/datalad/actions/workflows/test_extensions.yml)
-   [![Linters](https://github.com/datalad/datalad/actions/workflows/lint.yml/badge.svg)](https://github.com/datalad/datalad/actions/workflows/lint.yml)
```
Comment (yarikoptic) [inline README.md:39]: oh -- I think we managed to forget and manually introduce changes to README.md ... I will have to recover from that in this PR

**Pull request merged**